site/en/install/_index.yaml

```
@@ -39,7 +39,7 @@ landing_page:
     - heading: Download a package
       description: >
         <p>Install TensorFlow with Python's <em>pip</em> package manager.</p>
-        <aside class="note">TensorFlow 2 packages require a <code>pip</code> version >19.0.</aside>
+        <aside class="note">TensorFlow 2 packages require a <code>pip</code> version 20.3+ when running on macOS 11, or >19.0 on all other systems.</aside>
```

```suggestion
        <aside class="note">TensorFlow 2 packages require a <code>pip</code> version >20.3 when running on macOS 11, or >19.0 on all other systems.</aside>
```

Changing for consistency